javascript-light-bulb/main.js

```diff
@@ -0,0 +1,16 @@
+var button = document.querySelector('.button');
+var circle = document.querySelector('.circle');
+var body = document.querySelector('.body');
+button.addEventListener('click', click);
+
+function click(event) {
+  if (button.className === 'button button-off') {
```

Comment:
Instead of basing your functionality around the current classes on the DOM element, I would use a variable to keep track of if your lightbulb is "on" or "off". A boolean value would probably be a good choice. Remember what we discussed in Q&A, yesterday, we want all JavaScript functionality to not be reliant on data stored in the DOM as much as possible, you should have the information you need to do your work all centralized in your JavaScript.